### DISCUSSION TITLE
capacity not updated

### DISCUSSION THREAD

**Discussion by yuchen-sun** [juicedata/juicefs#939] [General]
**What happened**:
When updates capacity for juicefs, the capacity of mount point does not updated.
**What you expected to happen**:
use `df -h` can get new capacity 
**How to reproduce it (as minimally and precisely as possible)**:
```sh
df -h
update juicefs capacity
df -h
```
**Anything else we need to know?**

**Environment**:
- JuiceFS version (use `./juicefs --version`) or Hadoop Java SDK version:
v0.17.0
- Cloud provider or hardware configuration running JuiceFS:
- OS (e.g: `cat /etc/os-release`):
centos7.9
- Kernel (e.g. `uname -a`):
3.10.0-1160.el7.x86_64
- Object storage (cloud provider and region):
minio
- Redis info (version, cloud provider managed or self maintained):
- Network connectivity (JuiceFS to Redis, JuiceFS to object storage):
- Others:

**Comment by SandyXSD** [comment]
It takes at most one minute for mounted clients to recognize the change.
See: https://github.com/juicedata/juicefs/blob/main/pkg/meta/redis.go#L2077-L2079
